episodes/chapter4.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Using UI"
+teaching: 2
+exercises: 3
+---
+
+:::::::::::::::::::::::::::::::::::::: questions
+
+- What are the administrative entities in Vantage6?
+- What is the relation between the entities in Vantage6?
+- What is the permission system in Vantage6?
+- What are the default roles in Vantage6?
+- How to create a new organization using Vantage6 UI?
+- How to create a new user using Vantage6 UI?
+- How to create a new collaboration using Vantage6 UI?
+
+:::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Understand the types of administrative entities in Vantage6
+- Understand the relation between the administrative entities of Vatange6
+- Understand the permission system of Vantage6
+- Understand the default roles in Vantage6
+- Be able to create a new organization using the Vantage6 UI
+- Be able to create a new user using the Vantage6 UI
+- Be able to create a new collaboration using the Vantage6 UI
+- Be able to reset an api key for a node
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Vantage6 entities and their relations
+
+There are several entities in Vantage6, such as users, organizations, tasks, etc. These entities are created by users that have sufficient permission to do so and are stored in a database that is managed by the central server.
+
+The administrative entities in Vantage6 are:
+
+- **User**: a person who can perform certain actions in Vantage6.
+    - A user can only belong to one **organization**.
+    - A user can have multiple **roles**.
+    - A user can create **tasks** for one or more **organizations** within a **collaboration**.
+- **Organization**: an orgarnization to which users belong
+    - An organization can have zero and more **users**.
+    - An organization can have multiple **nodes**.
+- **Collaboration**: a collection of one or more **organizations**.
+    - Each participating **organization** needs a **node** to compute **tasks** for the **collaboration**.
```

Comment:
Maybe clarify: "So if a collaboration has 3 organizations in it, there are also 3 nodes: one for each organization. If one of the organizations participates in another collaboration, they need a separate node for that collaboration" (this is often something people struggle with at the start)